docs/sources/static/configuration/integrations/integrations-next/app-agent-receiver-config.md

```diff
@@ -2,10 +2,10 @@
 aliases:
 - ../../../../configuration/integrations/integrations-next/app-agent-receiver-config/
 canonical: https://grafana.com/docs/agent/latest/static/configuration/integrations/integrations-next/app-agent-receiver-config/
-title: app_agent_receiver_config
+title: app_agent_receiver_config revamp
 ---
 
-# app_agent_receiver_config
+# app_agent_receiver_config revamp
```

Comment:
Whats the reasoning for the revamp tag here and elsewhere? This one likely shouldnt have revamp since it did not exist in the original v1 integrations

Reply:
@mattdurham It's a suggestion for how we can fix up the duplicate titles. The duplicate is `blackbox config`. If I make a title change there, I need to make sure the change is consistent across all "next" topics.  If you have an idea for how we can fix up the duplicated blackbox-config title... :-)

Reply:
Thoughts on using the term `next` since the feature flag is `integrations-next`?

Reply:
Yes that works for me. Updated to next in a new commit.